tuist/tuist
# Per platform scheme

### Short description 📝
Pre multi-platform support, Tuist generated per-platform workspace schemes by default. This is no longer necessary since Xcode's build process can filter the targets to build based on the destination. I'm adjusting the logic that auto-generates the schemes to generate a single scheme containing all the targets instead of one per platform.

**Since this is a breaking change, I'm merging it into the `tuist-4` branch**

### How to test the changes locally 🧐
If you run Tuist a multi-platform project, for example, the one under `fixtures/multiplatform_app_with_sdk`, you should see only one auto-generated scheme.

### Contributor checklist ✅


### Reviewer checklist ✅
